website/docs/r/kusto_cluster.html.markdown

```diff
@@ -60,6 +62,13 @@ A `sku` block supports the following:
 
 * `capacity` - (Required) Specifies the node count for the cluster. Boundaries depend on the sku name.
 
+---
+
+A `identity` block supports the following:
+
+* `type` - (Required) The Managed Service Identity Type of this Kusto Cluster. Possible values is `SystemAssigned` (where Azure will generate a Service Principal for you).
+
+~> **NOTE:** When `type` is set to `SystemAssigned`, identity the Principal ID can be retrieved after the cluster has been created. More details are available below. See [documentation](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview) for additional information.
```

Comment:
Looks like we are missing some of the attributes that are computed. Can we get those added below?